lib/Util.ts

```diff
@@ -369,6 +369,30 @@ class Util {
 		}
 	}
 
+	public static incrementName(name: string, baseLength: number): string {
+		const text: string = name.slice(0, baseLength);
+		const number: number = parseInt(name.slice(baseLength), 10) || 0;
+		return `${text}${number + 1}`;
```

Comment:
Let's make the generator with space - app 1, app 2, Bullet Graph 1, Bullet Graph 2

Reply:
done